blobit-cli/src/main/java/org/blobit/cli/CommandGet.java

```diff
@@ -54,15 +57,28 @@ public void execute() throws Exception {
         }
         doWithClient(client -> {
             try (OutputStream ii = new BufferedOutputStream(new FileOutputStream(file))) {
-                DownloadPromise stat = client.getBucket(bucket)
-                        .downloadByName(name, (l) -> {
-                        }, ii, 0, -1);
-                System.out.println("FOUND OBJECT ID:" + stat.id + ", size " + stat.length + " bytes");
-                stat.get();
-                long _stop = System.currentTimeMillis();
-                double speed = (file.length() * 1000 * 60 * 60.0) / (1024 * 1024.0 * (_stop - _start));
-                System.out.println("OBJECT DOWNLOADED SUCCESSFULLY, " + speed + " MB/h");
+                if (name != null) {
```

Comment:
If provided both name and id it will default to name but it isn't clear.
IMHO it should fail and complain about it

Reply:
done